aqua-ethos-setup/aqua_ethos.sh

```diff
@@ -12,6 +12,7 @@ function setup {
 	if [[ -z "$HC_DIR" ]]; then log "HC_DIR environment variable required. Exiting..." && exit 1; fi
 	if [[ -z "$PASSWORD" ]]; then log "PASSWORD environment variable required. Exiting..." && exit 1; fi
 	if [[ -z "$FD_PASSWORD" ]]; then log "FD_PASSWORD environment variable required. Exiting..." && exit 1; fi
+	if [[ -z "$DATADOG_PASSWORD" ]]; then log "DATADOG_PASSWORD environment variable required. Exiting..." && exit 1; fi
```

Comment:
I'm not sure that we should require the DD password. Not everyone uses it. Let's skip this check here.